ontoconv/ontoflow.py

```diff
@@ -26,23 +149,38 @@ def parse_ontoflow(workflow_data, kb, outdir="."):
         Pipeline and workchain files are saved as yaml.
 
     """
-    outdir = Path(outdir)
-    # Fetch the pipeline specifications
-    pipelines = get_all_pipelines_spec(data=workflow_data)
```

Comment:
@louisponet 
In order to recognise when thedata comes from an already existing data set or not, a reorganisation of the pipeline was done with the get_all_pipelines_spec function. Now these are lost.